openstack_tasks.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Openstack tasks
+  hosts: "{{ osp_tasks_host | default('undercloud') }}"
```

Comment:
Is this a host group  that should be in `infrared workspace inventory`?
If so, can you add a note to the  README, to say which host groups should be in your inventory (it's already present)
For the stf functional tests for collectd and qdr, the suggested command in README.rst  is
`    ansible-playbook -i `infrared workspace inventory` stf_functional_tests.yml --tags "<some tag>"`

Reply:
Sure,I will add a note.

Reply:
osp_tasks_host  is just  an ansible variable. It can be override with --extra-vars. It has no effect except the case when you do want to override it. For example use different hosts.